packages/paste-website/src/components/tokens-list/token-card/index.tsx

```diff
@@ -7,40 +7,86 @@ import {Tooltip, useTooltipState} from '@twilio-paste/tooltip';
 import {rgbToHex} from '../../../utils/rgbToHex';
-import {BackgroundColor} from './preview/BackgroundColor';
+import {TokenExample} from './token-example';
+import type {Token} from '../types';
+import {SITE_BREAKPOINTS} from '../../../constants';
 
-interface TokenExampleProps {
-  category: string;
-  name: string;
-  value: string;
-}
-const PreviewComponent: React.FC<TokenExampleProps> = ({category, name, value}) => {
+const getTokenAltValue = ({category, value}: {category: string; value: Token['value']}): string | null => {
   switch (category) {
     case 'background-colors':
-      return <BackgroundColor name={name} value={value} />;
+    case 'border-colors':
+    case 'text-colors':
+      return rgbToHex(value);
+    case 'font-sizes':
+    case 'line-heights':
+    case 'sizings':
+    case 'spacings':
+      return remToPx(value, 'string') as string;
     default:
-      return <Box>{value}</Box>;
+      return null;
   }
 };
 
-export interface TokenCardProps {
-  category: string;
-  name: string;
-  value: string;
-  comment: string;
-}
-export const TokenCard: React.FC<TokenCardProps> = ({category, name, value, comment}) => {
+const TokenCardContent = styled.dl`
+  flex: 1;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr;
+  border-left: ${themeGet('borderWidths.borderWidth10')} solid ${themeGet('borderColors.colorBorderWeaker')};
+  padding: ${themeGet('space.space50')};
+  margin: ${themeGet('space.space0')};
+
+  @media screen and (min-width: ${SITE_BREAKPOINTS[0]}) {
+    padding: ${themeGet('space.space70')} ${themeGet('space.space60')};
+  }
+`;
+
+const TokenCardName = styled.dt`
+  display: flex;
+  align-items: center;
+  align-self: center;
+`;
+
+const TokenCardValue = styled.dd`
+  grid-row: 3;
+  margin: ${themeGet('space.space0')};
+
+  @media screen and (min-width: ${SITE_BREAKPOINTS[0]}) {
+    grid-row: 1/3;
+    grid-column: 2/2;
+    display: flex;
+    vertical-align: center;
+    margin-top: ${themeGet('space.space0')};
+    text-align: right;
+  }
+`;
+
+const TokenCardComment = styled.dd`
+  grid-row: 2;
+  margin: ${themeGet('space.space0')};
+`;
+
+export const TokenCard: React.FC<{
+  category: Token['category'];
+  name: Token['name'];
+  value: Token['value'];
+  backgroundColor: Properties['backgroundColor'];
+  comment?: Token['comment'];
+  useCamelCase?: boolean;
+}> = ({category, name, value, comment, backgroundColor, useCamelCase}) => {
   const tooltipState = useTooltipState();
   const [tooltipText, setTooltipText] = React.useState('Copy token name');
   // Prevents tooltip being visible on first render due to reakit positioning bug code
   const isFirstRender = React.useRef(true);
   const clipboard = useClipboard({copiedTimeout: 2000});
-  const camelCaseName = camelCase(name.replace('$', ''));
 
   const handleCopyName = React.useCallback(() => {
-    clipboard.copy(camelCaseName);
-  }, [camelCaseName]);
+    clipboard.copy(name);
```

Comment:
@TheSisb there was a small change since your review here but we are controlling the format of the displayed token name through a prop (useCamelCase) and the clipboard should copy whichever format is showing.